### ISSUE TITLE
[BUG] New installation of Kodi 19 and Seren 2.0.16

### ISSUE THREAD

**Issue by pogarek** [nixgates/plugin.video.seren#546]
This happens on fresh new installation of Kodi and Seren on Windows 10. 

**Logs**

```
2021-03-08 12:08:39.297 T:55064   ERROR <general>: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
                                                    - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                                   Error Type: <class 'SyntaxError'>
                                                   Error Contents: invalid syntax (UserDict.py, line 167)
                                                   Traceback (most recent call last):
                                                     File "C:\_MY_DATA_\TEMP\__KODI__\portable_data\addons\plugin.video.seren\service.py", line 9, in <module>
                                                       from resources.lib.common import tools
                                                     File "C:\_MY_DATA_\TEMP\__KODI__\portable_data\addons\plugin.video.seren\resources\lib\common\tools.py", line 17, in <module>
                                                       from resources.lib.third_party import pytz, tzlocal
                                                     File "C:\_MY_DATA_\TEMP\__KODI__\portable_data\addons\plugin.video.seren\resources\lib\third_party\pytz\__init__.py", line 32, in <module>
                                                       from resources.lib.third_party.pytz.lazy import LazyDict, LazyList, LazySet
                                                     File "C:\_MY_DATA_\TEMP\__KODI__\portable_data\addons\plugin.video.seren\resources\lib\third_party\pytz\lazy.py", line 3, in <module>
                                                       from UserDict import DictMixin
                                                     File "C:\_MY_DATA_\TEMP\__KODI__\system\python\Lib\UserDict.py", line 167
                                                       raise TypeError, "pop expected at most 2 arguments, got "\
                                                                      ^
                                                   SyntaxError: invalid syntax
                                                   -->End of Python script error report<--
                                                   
2021-03-08 12:08:39.307 T:55064    INFO <general>: Python interpreter stopped
2021-03-08 12:08:39.474 T:47908   ERROR <general>: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
                                                    - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                                   Error Type: <class 'ModuleNotFoundError'>
                                                   Error Contents: No module named 'socketserver'
                                                   Traceback (most recent call last):
                                                     File "C:\_MY_DATA_\TEMP\__KODI__\portable_data\addons\plugin.video.youtube\resources\lib\startup.py", line 11, in <module>
                                                       from youtube_plugin.kodion import service
                                                     File "C:\_MY_DATA_\TEMP\__KODI__\portable_data\addons\plugin.video.youtube\resources\lib\youtube_plugin\kodion\__init__.py", line 18, in <module>
                                                       from .abstract_provider import AbstractProvider
                                                     File "C:\_MY_DATA_\TEMP\__KODI__\portable_data\addons\plugin.video.youtube\resources\lib\youtube_plugin\kodion\abstract_provider.py", line 20, in <module>
                                                       from . import utils
                                                     File "C:\_MY_DATA_\TEMP\__KODI__\portable_data\addons\plugin.video.youtube\resources\lib\youtube_plugin\kodion\utils\__init__.py", line 19, in <module>
                                                       from .http_server import get_http_server, is_httpd_live, get_client_ip_address
                                                     File "C:\_MY_DATA_\TEMP\__KODI__\portable_data\addons\plugin.video.youtube\resources\lib\youtube_plugin\kodion\utils\http_server.py", line 10, in <module>
                                                       from six.moves import BaseHTTPServer
                                                     File "C:\_MY_DATA_\TEMP\__KODI__\portable_data\addons\script.module.six\lib\six.py", line 92, in __get__
                                                       result = self._resolve()
                                                     File "C:\_MY_DATA_\TEMP\__KODI__\portable_data\addons\script.module.six\lib\six.py", line 115, in _resolve
                                                       return _import_module(self.mod)
                                                     File "C:\_MY_DATA_\TEMP\__KODI__\portable_data\addons\script.module.six\lib\six.py", line 82, in _import_module
                                                       __import__(name)
                                                     File "C:\_MY_DATA_\TEMP\__KODI__\system\python\Lib\http\server.py", line 102, in <module>
                                                       import socketserver
                                                   ModuleNotFoundError: No module named 'socketserver'
                                                   -->End of Python script error report<--
                                                   
2021-03-08 12:08:39.483 T:47908    INFO <general>: Python interpreter stopped
2021-03-08 12:08:42.607 T:50700    INFO <general>: CIRServerSuite::Process: failed to connect to irss, will keep retrying every 5 seconds
2021-03-08 12:08:43.643 T:22748    INFO <general>: Loading skin file: MyVideoNav.xml, load type: KEEP_IN_MEMORY
2021-03-08 12:08:43.729 T:16476    INFO <general>: initializing python engine.
2021-03-08 12:08:43.783 T:16476   ERROR <general>: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
                                                    - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                                   Error Type: <class 'SyntaxError'>
                                                   Error Contents: invalid syntax (UserDict.py, line 167)
                                                   Traceback (most recent call last):
                                                     File "C:\_MY_DATA_\TEMP\__KODI__\portable_data\addons\plugin.video.seren\seren.py", line 6, in <module>
                                                       from resources.lib.common import tools
                                                     File "C:\_MY_DATA_\TEMP\__KODI__\portable_data\addons\plugin.video.seren\resources\lib\common\tools.py", line 17, in <module>
                                                       from resources.lib.third_party import pytz, tzlocal
                                                     File "C:\_MY_DATA_\TEMP\__KODI__\portable_data\addons\plugin.video.seren\resources\lib\third_party\pytz\__init__.py", line 32, in <module>
                                                       from resources.lib.third_party.pytz.lazy import LazyDict, LazyList, LazySet
                                                     File "C:\_MY_DATA_\TEMP\__KODI__\portable_data\addons\plugin.video.seren\resources\lib\third_party\pytz\lazy.py", line 3, in <module>
                                                       from UserDict import DictMixin
                                                     File "C:\_MY_DATA_\TEMP\__KODI__\system\python\Lib\UserDict.py", line 167
                                                       raise TypeError, "pop expected at most 2 arguments, got "\
                                                                      ^
                                                   SyntaxError: invalid syntax
                                                   -->End of Python script error report<--
```

Seren fails to load after installation.

**To Reproduce**
Just install Kodi 19.0 , then Seren 2.0.16 and try to open Seren addon.
**Expected behavior**
It works

**Kodi Version: 19.0 (downloaded today).

**Comment by jctarvin** [comment]
I started receiving this exact error this morning (3/11) on my system (ProBox Air 2 Android 6). Had been working just fine for the last month. Can no longer access my Trakt collection. Always throws this error. Re-sync'd and even rebuilt the Trakt database.  This is definitely a bug within Kodi 19/Seren 2.0.16. Hope you can fix it soon!! Thanks!!

**Comment by drinfernoo** [comment]
This should be fixed in 2.0.17.

**Comment by jctarvin** [comment]
Thanks! But, I've rolled back to Leia 18.9 which was working fine until Google Play auto-updated Kodi to 19 Matrix. That won't happen again. All ok now!

And, while I do use Real Debrid, I'm used to downloading the files to my local hard drive and being able to delete after watching without going to the Real Debrid site.

⁣John​


-------- Original Message --------
